backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -1,169 +1,103 @@
-// Navigation
-//---------------------------------------------------
-.inline-menu {
-  margin: 0;
-  -webkit-margin-before: 0;
-  -webkit-padding-start: 0;
+.sidebar {
+  @include display(flex);
+  @include flex-direction(column);
+  border-right: 1px solid $color-sidebar-border;
+  background: $color-sidebar-bg;
 }
 
-nav.menu {
+.admin-menu {
+  @include flex(1);
+  text-transform: uppercase;
+
   ul {
+    margin: 0;
+    padding: 0;
     list-style: none;
+  }
 
-    li {
-      a {
-        padding: 10px 0;
-        display: block;
-        position: relative;
-        text-align: left;
-        border: 1px solid transparent;
-        text-transform: uppercase;
-        font-weight: $font-weight-bold;
-        font-size: 90%;
-      }
-
-      &.active a {
-        color: $color-2;
-        border-left-width: 0;
-        border-bottom-color: $color-2;
-      }
+  li {
+    background: $color-navbar-bg;
 
-      &:hover a {
-        color: $color-2;
-      }
+    &:hover,
+    &.selected {
+      background: $color-navbar-active-bg;
     }
-  }
-}
 
-[data-hook="admin_login_navigation_bar"] {
-  ul {
-    text-align: right;
+    &.selected li {
+      padding-left: 2em;
+    }
 
-    li {
-      padding: 5px 0 5px 10px;
-      text-align: right;
-      font-size: 90%;
-      color: $color-link;
-      margin-top: 8px;
-
-      &[data-hook="user-logged-in-as"] {
-        width: 50%;
-        color: $color-body-text;
-      }
+    &:not(.selected):not(:hover) > ul {
+      display: none;
+    }
 
-      &.fa:before {
-        padding-right: 4px;
-      }
+    &:not(.selected) {
+      position: relative;
 
-      &:hover {
-        i {
-          color: $color-2;
-        }
+      // flyout nav
+      > ul {
+        position: absolute;
+        z-index: $z-index-navbar-flyout;
+        top: 0;
+        left: 100%;
+        width: $width-sidebar-flyout;
+        margin-left: 0;
+        padding: 1em 0;
+        border: 1px solid $color-sidebar-border;
+        background: $color-navbar-submenu-bg;
+        box-shadow: $box-shadow;
+        @include caret($direction: left, $color-caret: $color-navbar-submenu-bg);
+
+      &:before {
+        z-index: $z-index-navbar-flyout + 1;
+        top: 2em;
       }
+     }
     }
   }
-}
-
-#admin-menu {
-  background-color: $color-3;
 
-  ul{
-    display: table;
-    table-layout: fixed;
+  a {
+    display: inline-block;
```

Comment:
If it's `inline-block` and `width: 100%` we should just be able to one-line this with `display: block`.